### PR TITLE
Styling/cleanup

### LINKED ISSUE #4: Setup Responsiveness

### LINKED ISSUE #23: Responsive Design

### PR DESCRIPTION
**Pull request by tenthwalker**
What I did:
This pull request DRYs up styling between component files, condenses ids with similar styling needs into classes, and cleans up some missing semicolons/spacing in the code.

Did this break anything?

- [x] No
- [ ]  Yes

Type of change

- [x] New feature (non-breaking change which adds functionality)
- [x]  Styling 
- [ ]  Bug fix (non-breaking change which fixes an issue)
- [x]  Refactor(DRY-ing up/ reorganizing code, etc.)
- [ ]  Super small fix (Corrected a typo, removed a comment, etc.)
- [ ]  Skip all the other stuff and briefly explain the fix.

Checklist:

- [ ]  If this code needs to be tested, all tests are passing
- [x]  The code runs locally
- [ ]  There are comments where clarification/ organization is needed
- [x]  The code is DRY. If it's not, I tried my best
- [x]  I reviewed my code before pushing

What's next?
Accessibility audit

### CODE REVIEW

**Review by JoeyStewart — APPROVED** [review]

**Comment by JoeyStewart** [comment]
Nice Dry code! Looks much cleaner than before

File: src/App/App.css [hunk 1]
```diff
@@ -1,50 +1,46 @@
...
-  color: #161925;
 }
 
 h1 {
   margin: .67rem;
   font-size: 4rem;
 }
 
+section {
+  display: flex;
```
Comment (MaryBruff) [inline src/App/App.css:11]: This looks so much better with the column flex on it!

File: src/App/App.js [hunk 2]
```diff
@@ -7,19 +7,16 @@ import './App.css';
 import Card from '../Card/Card.js';
 import useSearchResults from '../hooks/useSearchResults.js';
 
-
 function App() {
   const { isLoading } = useAuth0();
-  
-  const randomSearchInputs = ['4chan', 'Titan (Submersible)', 'Billy Mitchell (gamer)', 'You Showed Me', 'Ezra Miller', 'Russel Brand', 'Bernie Madoff', 'Amy Winehouse']
+  const randomSearchInputs = ['4chan', 'Titan (Submersible)', 'Billy Mitchell (gamer)', 'You Showed Me', 'Ezra Miller', 'Russell Brand', 'Bernie Madoff', 'Amy Winehouse'];
```
Comment (MaryBruff) [inline src/App/App.js:12]: That extra 'L' game changer c;

File: src/Card/Card.css [hunk 3]
```diff
@@ -3,6 +3,7 @@
   max-width: 40%;
   border-bottom: .5rem solid #F1D302;
   border-right: .5rem solid #F1D302;
+  border-radius: 5px;
```
Comment (MaryBruff) [inline src/Card/Card.css:6]: Chef's kiss

File: src/WikipediaPage/WikipediaPage.js [hunk 4]
```diff
@@ -2,57 +2,54 @@ import React, { useState, useEffect } from 'react';
...
+  const url = `https://en.wikipedia.org/w/api.php?action=query&origin=*&prop=extracts&format=json&exintro=&titles=${pageTitle}`;
+
+  const extractAPIContents = (json) => {
+    const { pages } = json.query;
+    return Object.keys(pages).map((id) => pages[id].extract);
+  };
+
+const getContents = () => {
+  setLoading(true);
+  fetch(url)
```
Comment (MaryBruff) [inline src/WikipediaPage/WikipediaPage.js:17]: This looks so more dry and cleaner and honestly easier to read - great work!

**Comment by MaryBruff** [comment]
Most excellent! Really like the subtle changes of adding the border radius and flex column! Also code looks a lot better!

**Pull request merged**